### DISCUSSION TITLE
.Net8 vs .Net7

### DISCUSSION THREAD

**Discussion by PradeepPappuAto** [dotnet/maui#15631] [General]
I have few questions since the last 2 releases and I was wondering if some of my questions stated below could be answered

1. Future planned MAUI release dates (for .Net7 and .Net8)
2. Any links to board which details the issues being fixed for .Net7 and for .Net8 separately?
3. In the last few releases, more issues are being addressed in .Net8 than .Net7? Why is this so? when .Net8 is still in preview.

**Comment by milklike** [comment]
In my perspective，most issues are caused by winui3,other platform maybe have other problems

**Reply by PradeepPappuAto** [reply]
Shouldnt it be a parallel stream of work till .Net8 becomes a GA

**Comment by ziomek64** [comment]
> I have few questions since the last 2 releases and I was wondering if some of my questions stated below could be answered
> 
> 1. Future planned MAUI release dates (for .Net7 and .Net8)
> 2. Any links to board which details the issues being fixed for .Net7 and for .Net8 separately?
> 3. In the last few releases, more issues are being addressed in .Net8 than .Net7? Why is this so? when .Net8 is still in preview.

Because they are already preparing for .net 8 so they don't care that much for 7 which is an issue because this cycle repeats each year instead of changing .net every two years

**Reply by PradeepPappuAto** [reply]
Preparation for .Net8 is Okay.. but I think its too early as it isnt yet a RC.

We are planning to release the apps (upgraded from Xamarin.Forms, MVVM Cross & UWP) in August. We have hit some issues and have raised here. Those issues are being addressed in .Net8 and not in .Net7. As .Net8 GA is scheduled for November, we cant adopt to .Net8.

Any suggestions?

**Reply by MissedSte4k** [reply]
I wonder too, if it would be possible to at least push all/majority of the issues solved in current .net 8 to .net 7, because our .net 7 app is unreleasable right now after one of the .net 7 patches that introduced blank page bug in our app (app works fine again in .net 8). 
I had similar issues about a year back, where new patches broke our app, but then - I just opted to update to .net 7 preview, because we still had a lot of time until our planned release and they pushed fixes way faster there.

My suggestion would be to to hope for the best, but for me -  I think I'll wait for about a month or so... And if there is no .net 7 patch that solves our issues - I'll start migrating our Maui project to Xamarin and wait until they finally fix all the major .net Maui problems by the time of .net 10 or so
